packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -242,7 +242,7 @@ export default decorate([
                   className='form-control'
                   name='host'
                   onChange={effects.linkState}
-                  pattern='^([^\\/]+)\\([^\\/]+)$'
+                  pattern='^[^\\/]+(\\[^\\/]+)*$'
```

Comment:
Can you explain this change?

Reply:
Mainly allow the `host` to not have a path behind it (to fix the issue of the support ticket). And I also removed the first pair of parenthesis that are useless (right?).

Reply:
This is not a path, it's the name of the share and, AFAIU, it cannot be optional.

Reply:
I may be wrong though.